setup_cardano_crypto.sh

```diff
@@ -2,10 +2,8 @@
 
 git submodule update --init --recursive && \
 cd js-cardano-wasm && \
-git checkout def5d9a0ebd69435d29e737c7e99ea04f692ab70 && \
-cd rust && \
-git checkout 8b441cbbeba32ada20d065399083a16d85a5fe29 && \
-cd .. && \
+git checkout 386cc1ebe9cba1a1bf90304a491221d6986cd865 && \
```

Comment:
can we keep the checkout for rust plz I would prefer to be deterministic to a specific commit like `f8d7ac102b569f8198f227029cf190c084554ba8` (latest one on this moment in master)